homeassistant/components/roborock/strings.json

```diff
@@ -433,11 +433,11 @@
       "name": "Go to position",
       "description": "Send the vacuum to a specific position.",
       "fields": {
-        "x_coord": {
+        "x": {
           "name": "X-coordinate",
           "description": ""
         },
-        "y_coord": {
+        "y": {
           "name": "Y-coordinate",
           "description": ""
```

Comment:
Why do we have an empty description? I think it would be valuable to have something here that tells you what this value is

Reply:
https://github.com/home-assistant/core/pull/133994#discussion_r1898036031

Reply:
If you have a suggestion... but I did not see it, hence leaving it empty to avoid unnecessary translations.

Reply:
Maybe it would be nice to explain what way it is, as in, higher X -> more to the front, higher Y -> go to the right? Or is that an incorrect assumption

Reply:
It's all coordinates with the base station as reference... Maybe we should set something there.

Reply:
"The x coordinate as seen from the base station of the vaccum"

I think something like this would also explain the user why it wouldn't work anymore if you moved the base station

Reply:
Has been updated so let me see what you think 👍